### ISSUE TITLE
Platform error media_player.smartvideohub - Integration 'smartvideohub' not found.

### ISSUE THREAD

**Issue by starstudio13** [jnimmo/hass-smartvideohub#1]
I get an error when installing it on a fresh home assistant instance. I installed the folders and files as it says in the instructions. Does HACS need to be installed or any other dependencies?

**Comment by RaZer0r** [comment]
no, but it might be the same error as i experienced where it was unable to load due to a newer python version.

can you post the output of the log?
